src/routes/notifications/v1/entities/__tests__/safe-registration.builder.ts

```diff
@@ -2,20 +2,29 @@ import { faker } from '@faker-js/faker';
 import type { IBuilder } from '@/__tests__/builder';
 import { Builder } from '@/__tests__/builder';
 import type { SafeRegistration } from '@/routes/notifications/v1/entities/safe-registration.entity';
+import type { UUID } from 'crypto';
+import { safeRegistrationSignatureBuilder } from '@/routes/notifications/v1/entities/__tests__/create-signature.builder';
+import { getAddress } from 'viem';
 
-export function safeRegistrationBuilder(): IBuilder<SafeRegistration> {
+export async function safeRegistrationBuilder(args: {
+  signaturePrefix: string;
+  uuid: UUID;
+  cloudMessagingToken: UUID;
+  timestamp: number;
+}): Promise<IBuilder<SafeRegistration>> {
```

Comment:
I don’t have a strong opinion here, but since this builder is not useful without `signatures`, it needs to include a signature from the start. Therefore, it doesn’t make sense to create a builder without a `signature` and then add the signature later using `.with`.